contrib/http_util/src/main/java/io/opencensus/contrib/http/util/HttpPropagationUtil.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.http.util;
+
+import io.opencensus.trace.propagation.TextFormat;
+
+/**
+ * Utility class to get all supported {@link TextFormat}.
+ *
+ * @since 0.11.0
+ */
+public class HttpPropagationUtil {
+
+  private HttpPropagationUtil() {}
+
+  /**
+   * Returns the Stack Driver format. See {@link CloudTraceFormat} for more information.
```

Comment:
CloudTraceFormat is not public so is not going to be in the doc. better to copy link and summary here. Here is a link https://cloud.google.com/trace/docs/support